src/sentry/pipeline/base.py

```diff
@@ -118,7 +118,7 @@ def __init__(
         )
         self.state = self.session_store_cls(request, self.pipeline_name, ttl=PIPELINE_STATE_TTL)
         self.provider_model = provider_model
-        self.provider = self.get_provider(provider_key)
+        self.provider = self.get_provider(provider_key, organization=organization)
```

Comment:
passing this here so in the identity pipeline, we can use the org to do the feature flag check. i made the super class `get_provider` take a **kwargs, which i know is smelly but ideally we will remove the flag soon and clean this up.